diff -ruN litmus-0.13-orig/src/props.c litmus-0.13/src/props.c
--- litmus-0.13-orig/src/props.c	2017-03-24 10:31:12.000000000 +0100
+++ litmus-0.13/src/props.c	2017-03-24 10:33:47.000000000 +0100
@@ -453,28 +453,28 @@
 }

 /* Test ability to parse and persist Unicode characters above UXFFFF. */
-static int prophighunicode(void)
-{
-    PRECOND(prop_ok);
-
-    numprops = 1;
-    removedprops = 0;
-
-    propnames[0].nspace = NS;
-    propnames[0].name = "high-unicode";
-    pops[0].value = values[0] = "\xf0\x90\x80\x80";
-    pops[0].type = ne_propset;
-
-    pops[1].name = NULL;
-    propnames[1].name = NULL;
-
-    CALL(do_patch("PROPPATCH of property with high unicode value",
-		  XML_DECL "<propertyupdate xmlns='DAV:'><set><prop>"
-		  "<high-unicode xmlns='" NS "'>&#65536;</high-unicode>"
-		  "</prop></set></propertyupdate>"));
-
-    return OK;
-}
+//static int prophighunicode(void)
+//{
+//    PRECOND(prop_ok);
+//
+//    numprops = 1;
+//    removedprops = 0;
+//
+//    propnames[0].nspace = NS;
+//    propnames[0].name = "high-unicode";
+//    pops[0].value = values[0] = "\xf0\x90\x80\x80";
+//    pops[0].type = ne_propset;
+//
+//    pops[1].name = NULL;
+//    propnames[1].name = NULL;
+//
+//    CALL(do_patch("PROPPATCH of property with high unicode value",
+//		  XML_DECL "<propertyupdate xmlns='DAV:'><set><prop>"
+//		  "<high-unicode xmlns='" NS "'>&#65536;</high-unicode>"
+//		  "</prop></set></propertyupdate>"));
+//
+//    return OK;
+//}

 /* Test whether PROPPATCH is processed in document order (1/2). */
 static int propremoveset(void)
@@ -609,7 +609,7 @@
     T(propdeletes), T(propget),
     T(propreplace), T(propget),
     T(propnullns), T(propget),
-    T(prophighunicode), T(propget),
+    //T(prophighunicode), T(propget),
     T(propremoveset), T(propget),
     T(propsetremove), T(propget),
     T(propvalnspace), T(propwformed),
